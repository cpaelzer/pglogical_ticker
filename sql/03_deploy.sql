SELECT pglogical_ticker.deploy_ticker_tables();
