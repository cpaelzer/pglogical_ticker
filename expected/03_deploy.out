SELECT pglogical_ticker.deploy_ticker_tables();
 deploy_ticker_tables 
----------------------
                   11
(1 row)

